spec/active_record/hierarchical_query_spec.rb

```diff
@@ -218,19 +218,6 @@ def assert_ordered_by_name_asc(&block)
         ).to match_array [child_4, child_5]
       end
     end
-
-    describe 'binding values' do
```

Comment:
It seems like the ``bind`` method is just gone from AR 5:

* https://github.com/rails/rails/blob/master/activerecord/lib/active_record/relation/query_methods.rb
* https://apidock.com/rails/ActiveRecord/QueryMethods/bind

So my conclusion was: Test is not necessary anymore :) I've also removed the ``bind`` method from the ``CHILD_SCOPE_METHODS`` in ``query.rb`` due to that. Other ideas?